css/main.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: digital;
+  src: url(fonts/digital-7.ttf);
+}
+
+
+
+body {
+    background: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+input {
+    background-color: lightgray;
+    height: 50px;
+    width: 700px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+    border-radius: 15px;
+    
+    font: 60px digital;
+    color: darkblue;
+    padding-left: 20px;
+}
+
+#keyboard {
```

Comment:
Use ids for javascript and classes for styling.

Reply:
If I address digit button tags from both css and js files, how should I name them? Is it something like the following?
<div id="digitButton" class="digit-button">7</div>
<div id="digitButton" class="digit-button">8</div>
<div id="digitButton" class="digit-button">9</div>